transforms/tabular/tabular-merger-plugin/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "polus-plugins-transforms-tabular-tabular-merger"
+version = "0.1.0"
+description = "Merge vaex supported tabular file format into a single merged file."
+authors = [
+"Nick Schaub <nick.schaub@nih.gov>",
+"Hamdah Shafqat abbasi <hamdahshafqat.abbasi@nih.gov>"
+]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[[tool.poetry.source]]
+name = "test"
+url = "https://test.pypi.org/simple/"
+secondary = true
+
+[tool.poetry.dependencies]
+python = ">=3.9"
+filepattern = "^2.2.7"
```

Comment:
Update this to `2.0.0`, and remove the test PyPI reference above.

Reply:
done